vizro-core/tests/unit/vizro/conftest.py

```diff
@@ -43,6 +42,4 @@ def dashboard(page1, page2):
 
 @pytest.fixture()
 def dashboard_prebuild(dashboard):
-    yield dashboard.pre_build()
-    del dash.page_registry["Page 1"]
-    del dash.page_registry["Page 2"]
+    dashboard.pre_build()
```

Comment:
Can we just add `vizro_app` as input of this fixture and then replace:
`@pytest.mark.usefixtures("vizro_app", "dashboard_prebuild")` 
with:
`@pytest.mark.usefixtures("dashboard_prebuild")`
everywhere?

Reply:
I think this may not work in all cases, as there is other places, e.g. `TestGetActionLoopComponents`, where there is a different dashboard built. In order to acheive the above, there is a bigger rework needed